src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -266,6 +276,16 @@ private PSApplicationGateway CreateApplicationGateway()
                 applicationGateway.EnableHttp2 = true;
             }
 
+            if (this.EnableFIPS.IsPresent)
+            {
+                applicationGateway.EnableFips = true;
+            }
+
+            if (this.Zones != null)
```

Comment:
@akshaysngupta should this be `this.Zone` ?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/c696e9c6a3125703d7b0690bf27440d672c6e503/specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/applicationGateway.json#L1643